roles/cifmw_cephadm/templates/ceph_rgw.yml.j2

```diff
@@ -29,11 +29,11 @@ spec:
     backend_service: rgw.rgw
     frontend_port: 8080
     monitor_port: 8999
```

Comment:
Use `ssl_options` from https://docs.ceph.com/en/quincy/radosgw/frontends/#options here to get the functional equivalent of the following:
```
ceph config-key set ssl_option no_sslv2:sslv3:no_tlsv1:no_tlsv1_1
```